### DISCUSSION TITLE
Is it necessary to install office package?

### DISCUSSION THREAD

**Discussion by jiunder** [jxlsteam/jxls#214] [Q&A] [answered]
Hi :

Is it need office installed when using jxls api to create or read an excel file?

Thanks

**Comment by SoltauFintel** [comment, marked as answer]
It is important to use only Microsoft Excel to create the _template_ files.

JXLS is based on Apache POI. It should be asked there whether POI works without Office.
I don't think Office needs to be installed. However, the end user needs an Excel-like viewer to view the result file.
